python/packages/autogen-core/docs/src/user-guide/autogenstudio-user-guide/faq.md

```diff
@@ -13,8 +13,48 @@ A: You can specify the directory where files are stored by setting the `--appdir
 
 ## Q: Can I use other models with AutoGen Studio?
 
-Yes. AutoGen standardizes on the openai model api format, and you can use any api server that offers an openai compliant endpoint. In the AutoGen Studio UI, each agent has an `model_client` field where you can input your model endpoint details including `model`, `api key`, `base url`, `model type` and `api version`. For Azure OpenAI models, you can find these details in the Azure portal. Note that for Azure OpenAI, the `model name` is the deployment id or engine, and the `model type` is "azure".
-For other OSS models, we recommend using a server such as vllm, LMStudio, Ollama, to instantiate an openai compliant endpoint.
+Yes. AutoGen standardizes on the openai model api format, and you can use any api server that offers an openai compliant endpoint.
+
+AutoGen Studio is based on declaritive specifications which applies to models as well. Agents can include a model_client field which specifies the model endpoint details including `model`, `api_key`, `base_url`, `model type`.
+
+An example of the openai model client is shown below:
+
+```json
+{
+  "model": "gpt-4o-mini",
+  "model_type": "OpenAIChatCompletionClient",
+  "api_key": "your-api-key"
+}
+```
+
+An example of the azure openai model client is shown below:
+
+```json
+{
+  "model": "gpt-4o-mini",
+  "model_type": "AzureOpenAIChatCompletionClient",
+  "azure_deployment": "gpt-4o-mini",
+  "api_version": "2024-02-15-preview",
+  "azure_endpoint": "https://your-endpoint.openai.azure.com/",
+  "api_key": "your-api-key",
+  "component_type": "model"
+}
+```
+
+Have a local model server like Ollama or vLLM that provide an OpenAI compliant endpoint? You can use that as well.
+
+```json
+{
+  "model": "gpt-4o-mini",
+  "model_type": "OpenAIChatCompletionClient",
+  "api_key": "your-api-key",
+  "base_url": "http://localhost:8000"
```

Comment:
We need to change the `model`. Also, add `model_capabilities`. The doc: https://microsoft.github.io/autogen/dev/reference/python/autogen_ext.models.openai.html#autogen_ext.models.openai.OpenAIChatCompletionClient

Reply:
> We need to change the model. Also, add model_capabilities. 

What do you mean by we need to change the model?

Also by model capabilities, do you mean adding it to the example spec?

Reply:
> What do you mean by we need to change the model?

Change the model name to `llama2.1:3b` for example.

> Also by model capabilities, do you mean adding it to the example spec?

Yes. Add the `model_capabilities` field to the example. Because this is required if the model name is not already built-in.